Sources/Core/Components/PlayerError.swift

```diff
@@ -31,3 +31,8 @@ public enum PlayerError: Error {
     case playbackStalled
     case bufferingFailed
 }
+
+public struct PlayerErrorInfo {
+    public let errorCode: Int
```

Comment:
struct is about error, so code is enough.

Reply:
Done. Changed properties